internal/engine/up_watch.go

```diff
@@ -131,6 +132,10 @@ func snsToManifestWatcher(ctx context.Context, st *store.Store, timerMaker timer
 			//defer close(errs)
 
 			filter := ignore.CreateFileChangeFilter(manifest)
+			configMatcher, err := manifest.ConfigMatcher()
```

Comment:
i wonder if we should bake this logic into CreateFileChangeFilter? so FileChangeFilter simply wouldn't filter out anything that matched a ConfigFile

Reply:
Yeah I thought `CreateFileChangeFilter` was used somewhere else, but I see now that it isn't. Will do.